mlrose/__init__.py

```diff
@@ -9,5 +9,5 @@
 from .fitness import (OneMax, FlipFlop, FourPeaks, SixPeaks, ContinuousPeaks,
                       Knapsack, TravellingSales, Queens, MaxKColor, 
                       CustomFitness)
-from .neural import NeuralNetwork, LinearRegression, LogisticRegression
+from .neural import NeuralNetworkClassifier, NeuralNetworkRegressor, LinearRegression, LogisticRegression
```

Comment:
Broken because you did not make a NeuralNetworkRegressor.

Reply:
yup, forgot to take it out; fixed.